web/src/inbox_ui.js

```diff
@@ -741,6 +742,23 @@ export function get_focused_row_message() {
     return {message};
 }
 
+function is_focus_row_a_header() {
+    const $all_rows = get_all_rows();
+    const $focused_row = $($all_rows.get(row_focus));
+    if (is_row_a_header($focused_row)) {
+        return true;
+    }
+    return false;
+}
```

Comment:
I don't think you need this function; just have the caller check whether `inbox_message.message === undefined`.